### PR TITLE
Set up

### PR DESCRIPTION
**Pull request by FuadNabi**
## Project requirements

- Create a new Rails app called 'hello-rails-react'.
- Use the command rails new -j webpack my_new_rails_app
- Verify that the jsbundling-rails gem is added to your Gemfile.
- Initialize the project with Git.
- Install needed node dependencies
- Create a babel.config.js in the root folder of your project
- Update webpack.config.js to include the babel-loader
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table
- Create a static view that will be the root of your app.
- Create the App component with react-router and render it in your static view.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- Display the random greeting in your Greeting component.

### CODE REVIEW

**Comment by rloterh** [comment]
@FuadNabi 
Pull request description is empty! Kindly provide description to this Pull request

**Comment by FuadNabi** [comment]
> @FuadNabi Pull request description is empty! Kindly provide description to this Pull request

Hi @rloterh Thanks for letting me know about it. Fixed.

File: README.md [hunk 1]
```diff
@@ -50,23 +45,7 @@
...
-
-- [Budget Tracker](https://budget-tracker-azmz.onrender.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-### 📽️ Video Presentation <a name="video"></a>
-
-- [Video link](https://www.loom.com/share/1213a4eea2784cfe924a89b3ab8dd3ba)
+- **Add new greetings**
```
Comment (rloterh) [inline README.md:49]: - [ ] [OPTIONAL]
Kindly ensure that the _Getting Started_ section_ is filled with the appropriate instructions on how to _run_ your app, on _line 54_.

File: README.md [hunk 2]
```diff
@@ -129,7 +108,7 @@ To run the project, execute the following command:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/FuadNabi/budget-tracker/issues).
+Feel free to check the [issues page](https://github.com/FuadNabi/hello-rails-react/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rloterh) [inline README.md:114]: - [ ] [OPTIONAL]
Kindly ensure that the _Acknowledgement_ section_ is updated with the appropriate entities, on _line 125_. Apparently,[Gregoire Vella](https://www.behance.net/gregoirevella)'s design was not used on this project!

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Fuad,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾Congratulations!  🎉 💯 🌟 

## Highlights 🎯 
💠 Connection between Rails back-end and React front-end properly established in one app!
💠 Random greetings generating correctly!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- Please note that the Pull Request description should reflect the changes you implemented on this project rather than listing down project requirements.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**